src/styles/blocks/price.scss

```diff
@@ -0,0 +1,53 @@
+.price {
+  &__wrapper {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+
+    align-items: center;
+    margin-block: 24px 16px;
+    margin-inline: $margin-inline;
```

Comment:
The variable `$margin-inline` is used but not defined. Please define this variable to ensure the margin is applied correctly.